tests/spread/cross-compile/environment/snaps/target-arch/snap/snapcraft.yaml

```diff
@@ -21,3 +21,10 @@ environment:
 parts:
   my-part:
     plugin: nil
+    override-pull:
+      echo ${SNAPCRAFT_ARCH_TRIPLET}
```

Comment:
Might be worth checking that the values here are what we expect too

Reply:
Done.  I updated the spread test to confirm these variables are as-expected.